### PR TITLE
Policy info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -26,6 +26,30 @@ The following table provides component information for Anypoint Platform PCE:
 | API Testing	|   | X |
 |===
 
+For Anypoint Platform PCE Version 2.1, the following policies are included with API Manager:
```
Comment (federicoboerr) [inline modules/ROOT/pages/index.adoc:29]: The versions for the polices need to be specified. Tho I'm not sure ATM where to take them from.
Where did you get this info from?

**Review by federicoboerr — DISMISSED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/index.adoc:29]: Info in Slack: https://mulesoft.slack.com/archives/C27N99AM7/p1589457526489900

File: modules/ROOT/pages/upgrade.adoc [hunk 2]
```diff
@@ -83,6 +83,11 @@ $ kubectl get pods --all-namespaces
 ** xref:access-management-disclaimer.adoc[Configure a Disclaimer for Anypoint Private Cloud]
 ** xref:verify-nfs.adoc[NFS Server Prerequisites]
 
+[NOTE]
+When upgrading from a previous Anypoint Platform PCE version, all policies available in the previous version remain. Policies are not replaced, but additional policies might be included with the upgrade.
```
Comment (rddheath) [inline modules/ROOT/pages/upgrade.adoc:12]: Dangling participle in the first sentence: "When upgrading..., all policies available..."  All policies available are not doing the upgrading, right?

Comment (rddheath) [inline modules/ROOT/pages/upgrade.adoc:12]: "Policies are not replaced": isn't this redundant with "all policies in the previous version remain"? Or maybe I misunderstood. Does the paragraph mean "New versions of Anypoint Platform PCE always include the policies that are available in the previous version. However, new versions might add to the list of available policies."?

File: modules/ROOT/pages/upgrade.adoc [hunk 3]
```diff
@@ -83,6 +83,11 @@ $ kubectl get pods --all-namespaces
...
+
+Refer to xref:index.adoc[About Anypoint Platform PCE] for additional information about policies included with Anypoint Platform PCE.
```
Comment (rddheath) [inline modules/ROOT/pages/upgrade.adoc:14]: You write "for additional information about policies included with ...". However, it doesn't appear that you've given any information about policies included with PCE in the preceding paragraph.

**Review by rddheath — DISMISSED** [review]
Approved with comments.

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -26,6 +26,37 @@ The following table provides component information for Anypoint Platform PCE:
...
+| OpenID Connect OAuth 2.0 Token Enforcement | 1.3.1
+| PingFederate Access Token Enforcement | 1.3.1
+| Rate Limiting | 1.3.3
+| Rate limiting - SLA based | 1.2.5
+| Spike Control | 1.1.3
+| Tokenization | 1.1.0
+| XML Threat Protection | 1.1.3
+|===
+
+For additional information, refer to xref:api-manager::policies-landing-page.adoc[Policies].
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/index.adoc]: For additional information about these policies? That is, do they *need* to click the link?

**Review by Mulesoft-Mysti — DISMISSED** [review]

**Review by Mulesoft-Mysti — APPROVED** [review]

**Pull request merged**